pkgs/top-level/make-tarball.nix

```
@@ -55,6 +55,8 @@ pkgs.releaseTools.sourceTarball {
     echo "file json-br $packages" >> $out/nix-support/hydra-build-products
   '';
 
+  # --hard-dereference: reproducibility for src if auto-optimise-store = true
+  #   Some context: https://github.com/NixOS/nix/issues/10395
```

```suggestion
  #   Some context: https://github.com/NixOS/infra/issues/438
```

That gives better context imo (https://github.com/NixOS/infra/issues/438)

---

OK, why not.  I had been unsure which to choose when first writing that comment anyway.